src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -143,6 +144,9 @@
 
       <SourcesToCopyToTestDir Include="@(ContentWithTargetPath)" />
```

Comment:
I didn't really understand it either.  It seemed to change as part of the universal runner work.  I think the idea at a high level is we want to filter out stuff that will come from packages, but the rest of the content needs to be copied over.

If we can remove the CreateTestNuGetPackage stuff, that would be cool, I agree that this whole thing is kind of a mess and I'm unclear on what all we need to support.